StyleCop.Analyzers/StyleCop.Analyzers.Test.CSharp7/SpacingRules/SA1013CSharp7UnitTests.cs

```diff
@@ -49,7 +49,153 @@ public void TestMethod()
+        {
+            int* data1 = stackalloc[] { 1 , 1 } ;
+            int* data2 = stackalloc[] { 1 , 1 };
+            int* data3 = stackalloc[] { 1 , 1} ;
+            int* data4 = stackalloc[] { 1 , 1};
+            int* data5 = stackalloc[] { 1 , 1
+};
+            int* data6 = stackalloc[]
+            { 1 , 1};
+            int* data7 = stackalloc[]
+            {
+                1 , 1 } ;
+        }
+    }
+}
+";
+
+            var fixedCode = @"namespace TestNamespace
+{
+    public class TestClass
+    {
+        public unsafe void TestMethod()
+        {
+            int* data1 = stackalloc[] { 1 , 1 } ;
+            int* data2 = stackalloc[] { 1 , 1 };
+            int* data3 = stackalloc[] { 1 , 1 } ;
+            int* data4 = stackalloc[] { 1 , 1 };
+            int* data5 = stackalloc[] { 1 , 1
+};
+            int* data6 = stackalloc[]
+            { 1 , 1 };
+            int* data7 = stackalloc[]
+            {
+                1 , 1 } ;
+        }
+    }
+}
+";
+
+            DiagnosticResult[] expected =
+            {
+                Diagnostic().WithArguments(string.Empty, "preceded").WithLocation(9, 46),
+                Diagnostic().WithArguments(string.Empty, "preceded").WithLocation(10, 46),
+                Diagnostic().WithArguments(string.Empty, "preceded").WithLocation(14, 20),
+            };
+
+            await VerifyCSharpFixAsync(LanguageVersion.CSharp7_3, testCode, expected, fixedCode, CancellationToken.None).ConfigureAwait(false);
+        }
+
+        private static Task VerifyCSharpFixAsync(LanguageVersion languageVersion, string source, DiagnosticResult[] expected, string fixedSource, CancellationToken cancellationToken)
+        {
+            var test = new CSharpTest(languageVersion)
+            {
+                TestCode = source,
+                FixedCode = fixedSource,
+            };
+
+            if (source == fixedSource)
+            {
+                test.FixedState.InheritanceMode = StateInheritanceMode.AutoInheritAll;
+                test.FixedState.MarkupHandling = MarkupMode.Allow;
+                test.BatchFixedState.InheritanceMode = StateInheritanceMode.AutoInheritAll;
+                test.BatchFixedState.MarkupHandling = MarkupMode.Allow;
+            }
+
+            test.ExpectedDiagnostics.AddRange(expected);
+            return test.RunAsync(cancellationToken);
+        }
+
+        private class CSharpTest : StyleCopCodeFixVerifier<SA1013ClosingBracesMustBeSpacedCorrectly, TokenSpacingCodeFixProvider>.CSharpTest
+        {
+            public CSharpTest(LanguageVersion languageVersion)
+            {
+                this.SolutionTransforms.Add((solution, projectId) =>
+                {
+                    var parseOptions = (CSharpParseOptions)solution.GetProject(projectId).ParseOptions;
+                    return solution.WithProjectParseOptions(projectId, parseOptions.WithLanguageVersion(languageVersion));
+                });
+            }
```

Comment:
❗️ It would probably make sense for the future to move this functionality to the `StyleCopCodeFixVerifier<TAnalyzer, TCodeFix> class.